services/enrichment/manager/src/events/onFileIngested.js

```diff
@@ -45,13 +48,11 @@ export const handler = async (event, context, callback) => {
       })
       .promise();
 
-    const queueUrl = `https://sqs.${REGION}.amazonaws.com/${accountId}/${QUEUE_NAME}`;
-
     // Prepare push to SQS
     const queueStream = new QueueStream({
       objectMode: true,
-      sqs: new AWS.SQS({ apiVersion: '2012-11-05' }),
```

Comment:
Please update comment and naming of this stream correspondingly - it's not an SQS, but SNS.